java/compaction/compaction-status-store/src/main/java/sleeper/compaction/status/store/job/DynamoDBCompactionJobStatusFormat.java

```diff
@@ -197,7 +199,9 @@ private static ProcessStatusUpdate getStatusUpdate(Map<String, AttributeValue> i
                         .committedBySeparateUpdate(getBooleanAttribute(item, JOB_COMMITTED_SEPARATELY))
                         .build();
             case UPDATE_TYPE_COMMITTED:
-                return CompactionJobCommittedStatus.committedAt(getInstantAttribute(item, UPDATE_TIME));
+                return CompactionJobCommittedStatus.committedAt(
+                        getInstantAttribute(item, COMMIT_TIME),
+                        getInstantAttribute(item, UPDATE_TIME));
```

Comment:
It would be useful if the committedAt function was named to indicate the order of the parameters, maybe something like committedAtAndUpdatedAt.

Reply:
Thanks, I've renamed the static constructor to be consistent with the other static constructors for compaction job statuses